internal/http/types.go

```diff
@@ -87,9 +87,37 @@ type FluxNotifyRequest struct {
 	FluxEvent   event.Event
 }
 
+type DeploymentEvent struct {
+	Name          string `json:"name,omitempty"`
+	Namespace     string `json:"namespace,omitempty"`
+	AvailablePods int32  `json:"podCount,omitempty"`
```

Comment:
For consistency could we use `availablePods` as the JSON field? Further I think we should not omit empty values (`0`) just to better understand the payloads.

```suggestion
	AvailablePods int32  `json:"availablePods"`
```